YvesIraguha/lost-and-found-api
# Bg fix document search

#### What does this PR do?
Implement full-text search
#### Description of Task to be completed?
- Change search controller to do a full-text search
- Change search controller to return every document matching the string
#### How should this be manually tested?
- Clone this repo
- Switch your working directory
- Checkout this branch
- Build docker image: `docker-compose build`
- Start the container `docker-compose up`
- In postman send a get request on `http://localhost:3000/api/v1/search/documents` with the following
```json
{ "query": "searchText" }
````
> You should see any document having a field matching your searchText
#### Any background context you want to provide?
- You have to be signed in and your auth-token presented in the headers as auth-token
- You have to contain documents in your document otherwise you may need to create them before checking

#### Questions:
